provision/testdata/base_case_snat.kube.yaml

```diff
@@ -10,7 +10,27 @@ spec:
     plural: acicontainersoperators
     singular: acicontainersoperator
   scope: Namespaced
-  version: v1alpha1
+  versions:
+  - name: v1alpha1
+    served: true
+    storage: true
+    subresources:
+      status: {}
+    schema:
+      openAPIV3Schema:
+        description: acicontainersoperator owns the lifecycle of ACI objects in the cluster
+        properties:
```

Comment:
Don't we have to add before spec:
          apiVersion:
            type: string
          kind:
            type: string
          metadata:
            type: object

Reply:
likewise everywhere

Reply:
Posted these!